src/entities/CollectionToken/CollectionTokenTransformers.ts

```diff
@@ -1,10 +1,31 @@
-import { Erc721Token } from '../Erc721Token/Erc721Token';
+import { tokenKinds } from '@airswap/constants';
+import { TokenInfo } from '@airswap/typescript';
+
 import { CollectionToken } from './CollectionToken';
 
-export const transformErc721TokenToCollectionToken = (erc721Token: Erc721Token, tokenId: number, price: number): CollectionToken => ({
+export const transformErc721TokenToCollectionToken = (tokenInfo: TokenInfo, tokenId: number, price: number): CollectionToken => ({
+  id: tokenId,
+  image: (tokenInfo.extensions?.metadata as any).image.replace('ipfs://', 'https://ipfs.io/ipfs/'),
+  description: (tokenInfo.extensions?.metadata as any).description,
+  price,
+  name: (tokenInfo.extensions?.metadata as any).name,
+});
+
+export const transformErc1155TokenToCollectionToken = (tokenInfo: TokenInfo, tokenId: number, price: number): CollectionToken => ({
```

Comment:
Transform all the possible metadata properties to CollectionToken.

https://docs.opensea.io/docs/metadata-standards

Not everything is in there but by taking a look at some ERC1155 tokens you get the idea of what's standard:

![Screenshot 2023-02-05 at 06 20 17](https://user-images.githubusercontent.com/86911296/216803259-4b01073a-b2e1-4068-8d91-c4cb6f3b2464.png)
![Screenshot 2023-02-05 at 06 25 17](https://user-images.githubusercontent.com/86911296/216803261-0b2c916d-d50d-425b-8d0a-7adccf4afd6f.png)

Reply:
If there are different metadata standards then identify these standards and make entities for them.